package.json

```diff
@@ -47,7 +47,7 @@
     "leaflet-fullscreen": "^1.0.1",
     "leaflet-hash": "^0.2.1",
     "leaflet-i18n": "0.2.0",
-    "leaflet-loading": "^0.1.18",
+    "leaflet-loading": "ebrelsford/Leaflet.loading",
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12969424/16940763/911c806a-4dbe-11e6-8389-78674e7da462.png)

npm is not available with leaflet-loading v0.1.22, only 0.1.21.

![image](https://cloud.githubusercontent.com/assets/12969424/16940844/ff801f26-4dbe-11e6-94df-d7fe6836be87.png)

When npm has the version 0.1.22, you can set it then.